app/API/V1/Repositories/LoginAttemptRepository.php

```diff
@@ -152,36 +152,24 @@ private function getAttemptStatus(LoginAttempt $attempt)
 
         /** Check for attempts */
         if ($max_partial_lock && $attempt->getCount() >= $max_partial_lock) {
```

Comment:
@jerazo are all the account lock combinations tested? I see there was some big changes here, so I wanted to make sure they were all still supported, and working

Reply:
Yes, we have all the lock cases supported with such smaller code.